src/plugin.rs

```diff
@@ -41,20 +41,20 @@ use crate::{
 
 /// Sets for this plugin.
 #[derive(Debug, Hash, PartialEq, Eq, Clone, SystemSet)]
-pub enum Set {
+pub enum SvgSet {
     /// Set in which [`Svg2dBundle`](crate::bundle::Svg2dBundle)s get drawn.
-    SVG,
+    Svg, // TODO: do we need this? Would it be more useful to split the different parts of this set?
```

Comment:
Can you elaborate a bit?

Reply:
I guess, I can see `SvgSet` either naming all of the different systems as variants or just being a unit struct. I don't see a one-variant enum being useful. It wasn't a pertinent thing so I didn't make a decision, but it's something to think about.

Reply:
Ok, got it. I don't actually remember why i used a single variant enum, so it's fine by me to change it to a unit struct.